bin/inx-chronicle/src/api/config.rs

```diff
@@ -13,17 +17,62 @@ use super::error::ConfigError;
 pub struct ApiConfig {
     pub port: u16,
     pub allow_origins: Option<SingleOrMultiple<String>>,
+    pub password_hash: String,
+    pub password_salt: String,
+    pub jwt_salt: String,
+    #[serde(with = "humantime_serde")]
+    pub jwt_expiration: Duration,
+    pub public_routes: Vec<String>,
 }
 
 impl Default for ApiConfig {
     fn default() -> Self {
         Self {
             port: 9092,
-            allow_origins: Some(String::from("*").into()),
+            allow_origins: Some("*".to_string().into()),
+            password_hash: "c42cf2be3a442a29d8cd827a27099b0c".to_string(),
+            password_salt: "saltines".to_string(),
+            jwt_salt: "Chronicle".to_string(),
+            jwt_expiration: time::Duration::minutes(30).try_into().unwrap(),
```

Comment:
Is that really a sensible default? Shouldn't the expiration be optional too?

Reply:
If they never expire, then anyone who gets their hands on a token could use it forever. Not sure how important that is, but I'd rather default to expiring and let users decide if they want that risk.

Reply:
If we change the `config.template.toml` to include the expiration field (set to never expire) and write a comment we should be good. Securing Chronicle in production is not our job but rather has to be done individually by users — we just need to give them the tools they need to do so, together with documentation.

Reply:
There's no reason to let the token never expire (if we set the default to 24 hours it should be enough in almost all cases). Also ... it's just about not needing to re-authenticate oneself for subsequent requests. So why open a security hole for a convenience thing. Chronicle will just create a new token for them. And also... most people don't read comments carefully enough or understand some of the implications if they don't change the 'never expire' default.